lib/src/components/NxFilterInput/NxFilterInput.scss

```diff
@@ -38,7 +38,7 @@
     @include nx-button-helpers.small-icon-btn;
 
     margin: 0;
-    padding: calc(var(--nx-spacing-base) - 2px) 0;
+    padding: calc(var(--nx-spacing-base) - var(--nx-padding-offset-filter-clear-button)) 0;
```

Comment:
I don't think we need a new variable here.  We just need to take one additional pixel off of the top and bottom paddings produced by `nx-button-helpers.small-icon-btn`.  How about giving that mixin a parameter called `extra-vertical-padding` or something, and then passing `-1px` for it?

Reply:
> I don't think we need a new variable here. We just need to take one additional pixel off of the top and bottom paddings produced by `nx-button-helpers.small-icon-btn`. How about giving that mixin a parameter called `extra-vertical-padding` or something, and then passing `-1px` for it?

Thank you, that's a great idea. Changed here: ab33a4d82a32620ec803fe44ebbea22d815d66ca and e7eeb6339c172b9ffa6ae94b9fb9cc35f2058bcb